biz.aQute.bndlib/src/aQute/bnd/header/Parameters.java

```diff
@@ -250,4 +251,10 @@ public void mergeWith(Parameters other, boolean override) {
 	public boolean allowDuplicateAttributes() {
 		return allowDuplicateAttributes;
 	}
+
+	public static Collector<String,StringBuilder,Parameters> toParameters() {
```

Comment:
Here is a slightly simplified version using StringJoiner.

```java
public static Collector<String, StringJoiner, Parameters> toParameters() {
	return Collector.of(() -> new StringJoiner(","), StringJoiner::add, StringJoiner::merge,
		j -> new Parameters(j.toString()));
}
```

An alternate version avoids string concatenation:

```java
public static Collector<String, Parameters, Parameters> toParameters() {
	return Collector.of(Parameters::new, Parameters::accumulator, Parameters::combiner);
}
private static void accumulator(Parameters p, String s) {
	p.mergeWith(OSGiHeader.parseHeader(s), true);
}
private static Parameters combiner(Parameters t, Parameters u) {
	t.mergeWith(u, true);
	return t;
}
```